### DISCUSSION TITLE
After sending sats on-chain to another wallet - should I recover my inbound liquidity?

### DISCUSSION THREAD

**Discussion by Purchase73** [ACINQ/phoenix#549] [Q&A]
Or does the sending need to happen on lightning?

One other thing - I purchased inbound liquidity to get the amount I needed.

**Comment by dpad85** [comment]
When you send funds on-chain, the size of your channel is mechanically reduced: what you are doing is actually taking funds from the channel, and sending them to another address. That's why it's called a "splice-out".

So the capacity of the channel diminishes. The liquidity stays the same. You do not recover inbound liquidity.

When you send funds over Lightning, the channel's capacity is not affected, and you will recover liquidity.
